lib/Wikia/src/Service/User/Attributes/AttributesModule.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Wikia\Service\User\Attributes;
+
+use Wikia\DependencyInjection\InjectorBuilder;
+use Wikia\DependencyInjection\Module;
+use Wikia\Persistence\User\Attributes\AttributePersistence;
+use Wikia\Persistence\User\Attributes\AttributePersistenceSwagger;
+
+class AttributesModule implements Module {
+	public function configure( InjectorBuilder $builder ) {
+		$builder
+			->bind( AttributeService::class )->toClass( AttributeKeyValueService::class )
+			->bind( AttributePersistence::class )->toClass( AttributePersistenceSwagger::class )
+			->bind( UserAttributes::DEFAULT_ATTRIBUTES )->to( function() {
```

Comment:
When does this become relevant?  Does the ```@Inject({``` comment below get treated as an annotation somehow?